components/Drawer.tsx

```diff
@@ -2,95 +2,125 @@ import React from "react";
-              component="a"
-              href="https://advisorysg.typeform.com/to/Z1MHxaFj#source=mentorsite"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              <ListItemText
-                primary="Apply Now"
-                style={{ flexGrow: "0", marginRight: "0.5rem" }}
-              />
-              <ListItemIcon>
-                <LaunchIcon />
-              </ListItemIcon>
-            </ListItemButton>
-          </ListItem>
-        </List>
-      </div>
-    );
-  };
+  const LinkListDesktop = () => (
+    <List
+      style={{
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "flex-end",
+      }}
+    >
+      <ListItem disablePadding>
+        <a
+          href={URL_HOME}
+          onClick={handleDrawerToggle}
+          style={desktopLinkStyle}
+        >
+          Home
+        </a>
+      </ListItem>
+      <ListItem disablePadding>
+        <a
+          href={URL_MENTORS}
+          onClick={handleDrawerToggle}
+          style={desktopLinkStyle}
+        >
+          Find A Mentor
+        </a>
+      </ListItem>
+      <ListItem disablePadding>
+        <a href={URL_FAQ} onClick={handleDrawerToggle} style={desktopLinkStyle}>
+          FAQ
+        </a>
+      </ListItem>
+      <ListItem disablePadding>
+        <a
+          href={URL_APPLY}
+          onClick={handleDrawerToggle}
+          style={desktopLinkStyle}
+        >
+          Apply Now
+        </a>
+      </ListItem>
+    </List>
+  );
+
+  const LinkListMobile = () => (
+    <List
+      style={{
+        display: "block",
+        flexDirection: "column",
+        textAlign: "right",
+        paddingLeft: "62vw",
+      }}
+    >
+      <ListItem disablePadding style={{ paddingLeft: "60px" }}>
+        <a href={URL_HOME} onClick={handleDrawerToggle} style={mobileLinkStyle}>
+          Home
+        </a>
+      </ListItem>
+      <ListItem disablePadding style={{ paddingRight: "70px" }}>
+        <a
+          href={URL_MENTORS}
+          onClick={handleDrawerToggle}
+          style={mobileLinkStyle}
+        >
+          Find A Mentor
+        </a>
+      </ListItem>
+      <ListItem disablePadding style={{ paddingLeft: "70px" }}>
+        <a href={URL_FAQ} onClick={handleDrawerToggle} style={mobileLinkStyle}>
+          FAQ
+        </a>
+      </ListItem>
+      <ListItem disablePadding style={{ paddingLeft: "20px" }}>
+        <a
+          href={URL_APPLY}
+          onClick={handleDrawerToggle}
+          style={mobileLinkStyle}
+        >
+          Apply Now
+        </a>
+      </ListItem>
+    </List>
+  );
 
   return (
-    <Box sx={{ display: "flex" }}>
-      <AppBar style={{ background: "white" }} className="toolbar-component">
+    <Box sx={{ display: "flex", paddingBottom: "70px" }}>
+      <AppBar
+        style={{
+          background: "white",
+          boxShadow: "none",
+          position: "sticky",
+          zIndex: 1300,
```

Comment:
What's the reason behind choosing 1300?

Reply:
I tried using smaller values but the Drawer covered the logo and the X. This value allows the logo and X to be above the Drawer.